plugins/babelrc/file-preprocessor.js

```diff
@@ -4,6 +4,16 @@ const debug = require('debug')('cypress-react-unit-test')
 const webpackPreprocessor = require('@cypress/webpack-preprocessor')
 const { addImageRedirect } = require('../utils/add-image-redirect')
 
+const wpPreprocessorOptions = {
```

Comment:
Another question – do we need to keep using `defaultOptions`? Or we can just make our own

Reply:
I would not change unless necessary